### PR TITLE
Form section

### PR DESCRIPTION
**Pull request by KaskMIL**
# Form Section and Footer
## Footer contains Social Media Icons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -180,7 +180,38 @@ <h3> <img src="/vectorsImg/Tertiary/Icons/circle.svg" alt=""> Skills</h3>
...
     </main>
+    <footer>
+        <ul>
+            <li><a href="#"><i class="bi bi-github"></i></a></li>
+            <li><a href="#"><i class="bi bi-linkedin"></i></a></li>
+            <li><a href="#"><img src="vectorsImg/Tertiary/Icons/Vector.svg" alt="peace button social media"></a>
+            </li>
+            <li><a href="#"><i class="bi bi-twitter"></i></a></li>
+            <li><a href="#"><img src="vectorsImg/Tertiary/Icons/Mvector.svg" alt="m button social media"></a></li>
+        </ul>
```
Comment (Wes-Isaac) [inline index.html:213]: - [ ] Great job using a list to layout the icons, however, the styling is not like the design, kindly fix that.

### Your Implementation
image


### Figma's Design
image

File: index.html [hunk 2]
```diff
@@ -180,7 +180,38 @@ <h3> <img src="/vectorsImg/Tertiary/Icons/circle.svg" alt=""> Skills</h3>
...
+                        <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+                    </li>
+                    <li>
+                        <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+                    </li>
+                    <li>
+                        <textarea name="user_msg" id="msg" maxlength="500" placeholder="Enter text here" required></textarea>
+                    </li>
+                    <li class="btn-container">
+                        <button type="submit">Get in touch</button>
```
Comment (Wes-Isaac) [inline index.html:199]: - [ ] Great job using a button, please add button interaction hover, active (when it's pressed), disabled and enable (default state).

image

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @KaskMIL, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters are on point ✔
- Splendid work with the README file 👀
- Proper use of GitHub flow 👌

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by KaskMIL** [comment]
Hi @Wes-Isaac,
Thanks for your review, I think I could implement every change required.
👍

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @KaskMIL,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [x] Great job implementing the changes requested by the previous reviewer but you missed  the horizontal bar at the bottom of the media icons. I think it will be great if your media icons also had an horizontal bar beneath them like in the Template design

| Your Design  | Template Design |
| ------------- | ------------- |
| Screenshot from 2022-04-26 23-14-03 | Screenshot from 2022-04-26 23-17-13 |

> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by KaskMIL** [comment]
Hi @topeogunleye,
Thanks for the review, change implemented
cheers!

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @KaskMIL   , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Decriptive readme file.
:heavy_check_mark: Well implemented Figma design.

### Optional suggestions :arrow_down_small: 

- [ ] The commit message should be written in the imperative tense. `Ex: "Fix bug" instead of "Fixed bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

- [ ] In your `README` file, there is no "Getting Started" section. Kindly add the getting started section. This should be an example of how you may give instructions on setting up your project locally.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**